src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const bigArray = sourceString.split(';');
```

Comment:
Consider adding a check to ensure `sourceString` is not empty or null before splitting it. This will prevent potential errors when the input is not as expected.